docs/content/reference/command-line-interface/_index.md

```diff
@@ -162,8 +163,8 @@ In this section, you will find detailed explanations and usage examples for each
 **Purpose**: The `apply` command is used to create or update one or more resources described in a YAML file. The YAML file can contain multiple resource definitions and can be any mix of resource types.
 
 **Flags and Arguments**:
-- `-f|--files <file> <file> <...>`: Specifies the location of one or more YAML files containing the resources to create. File locations can be absolute or relative file paths or URLs.
-- `-n|--namespace <namespace>`: Specifies the namespace where the resources should be created. If not provided, the default namespace configured using the `drasi namespace set` command is used.
+- `-f|--files <file> <file> <...>` (required): Specifies the location of one or more YAML files containing the resources to create. File locations can be absolute or relative file paths or URLs.
```

Comment:
This is not required.  You can pipe yaml into the CLI like this:
```
echo "some yaml" | drasi apply
```

Reply:
I marked the flag as "required" keeping the examples already provided in mind. I can add a note saying that the flag is not required when a file name is passed in/piped in.

Reply:
Removing option/flag descriptions after offline discussion.